docs/pages/scrollboundarycontainer.js

```diff
@@ -121,48 +121,49 @@ function ScrollBoundaryContainerExample() {
               idealDirection="right"
               text="See stats about different types of content created by you and/or others on Pinterest. Filter to get more details on your organic (not an ad) and paid (promoted as an ad) content."
             >
-              <Icon
-                icon="info-circle"
-                accessibilityLabel="Information"
-                color="default"
-              />
+              <TapArea>
+                <Icon
+                  icon="info-circle"
+                  accessibilityLabel="Information"
+                  color="default"
+                />
+              </TapArea>
             </Tooltip>
           </Flex>
-          <Box
-            display="flex"
-            direction="column"
-          >
-          <Box paddingY={1}>
-              <RadioButton
-                checked={content === "all"}
-                id="allcontent"
-                label="All"
-                name="content"
-                onChange={() => setContent( "all" )}
-                value="all"
-              />
-            </Box>
-            <Box paddingY={1}>
-              <RadioButton
-                checked={content === "organic"}
-                id="organic"
-                label="Organic"
-                name="content"
-                onChange={() => setContent( "organic" )}
-                value="organic"
-              />
-            </Box>
-            <Box paddingY={1}>
-              <RadioButton
-                checked={content === "paid"}
-                id="paid"
-                label="Paid and earned"
-                name="content"
-                onChange={() => setContent( "paid" )}
-                value="paid"
-              />
-            </Box>
-          </Box>
+          <Flex direction="column">
+            <Fieldset legend="Select content type" legendDisplay="hidden">
+              <Box paddingY={1}>
```

Comment:
should this use Flex and `gap`, or does that not work well with Fieldset?

Reply:
DONE. good catch